build/generate-licenses.go

```diff
@@ -15,19 +17,22 @@ import (
 	"path/filepath"
 	"strings"
 
+	"code.gitea.io/gitea/modules/json"
 	"code.gitea.io/gitea/modules/util"
 )
 
 func main() {
 	var (
-		prefix         = "gitea-licenses"
-		url            = "https://api.github.com/repos/spdx/license-list-data/tarball"
-		githubApiToken = ""
-		githubUsername = ""
-		destination    = ""
+		prefix                    = "gitea-licenses"
+		url                       = "https://api.github.com/repos/spdx/license-list-data/tarball"
+		githubApiToken            = ""
+		githubUsername            = ""
+		destination               = ""
+		licenseAliasesDestination = ""
 	)
 
 	flag.StringVar(&destination, "dest", "options/license/", "destination for the licenses")
+	flag.StringVar(&licenseAliasesDestination, "aliasesdest", "options/license-aliases.json", "destination for same license json")
```

Comment:
`options` dir is a problematic place to store `license-aliases.json`

At least, it should be in something like `options/license/etc/license-aliases.json`

And it should only managed with `("dest", "options/license/")` together, do not customize it anymore.

Reply:
d26f2a0f3d8afba85488a6d21d19f177ff1e2153